src/dapps/ens-manager-dapp/TheENSManagerLayout.vue

```diff
@@ -61,7 +61,7 @@
     =====================================================================================
     -->
       <template #tabContent2>
-        <v-sheet min-height="500px" class="pa-12">
+        <v-sheet min-height="500px" class="pa-4 pa-md-12">
```

Comment:
can you double check pa-4? i think we are using pa-3 on xs screens

Reply:
fixed